packages/icons/resources/icons/icons.json

```diff
@@ -3538,5 +3538,11 @@
         "iconName": "array-timestamp",
         "tags": "array, timestamp",
         "group": "miscellaneous"
-    }
+    },
+    {
+        "displayName": "Layer Outline",
+        "iconName": "layer-outline",
+        "tags": "zone, level, outline",
+        "group": "interface"
+    },
```

Comment:
build is failing due to this trailing comma, which is not permitted in JSON at the end of a list (yes, it's annoying, since JS _does_ permit it 🤷‍♂️ )